src/style.css

```
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 2px;
+}
+
+.container {
+  width: 96px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
```

```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url('images/star-active.svg');
}
```
combine it with comma